rules/assertion-arguments.js

```diff
@@ -86,6 +99,113 @@ const expectedNbArguments = {
 	}
 };
 
+const actualExpectedAssertions = new Set([
+	'deepEqual',
+	'is',
+	'like',
+	'not',
+	'notDeepEqual',
+	'throws',
+	'throwsAsync'
+]);
+
+const relationalActualExpectedAssertions = new Set([
+	'assert',
+	'truthy',
+	'falsy',
+	'true',
+	'false'
+]);
+
+const comparisonOperators = new Map([
+	['>', '<'],
+	['>=', '<='],
+	['==', '=='],
+	['===', '==='],
+	['!=', '!='],
+	['!==', '!=='],
+	['<=', '>='],
+	['<', '>']
+]);
+
+const flipOperator = operator => comparisonOperators.get(operator);
+
+function isStatic(node) {
+	const staticValue = getStaticValue(node);
+	return staticValue !== null && typeof staticValue.value !== 'function';
+}
+
+function * sourceRangesOfArguments(sourceCode, callExpression) {
+	const openingParen = sourceCode.getTokenAfter(
+		callExpression.callee,
+		{filter: token => isOpeningParenToken(token)}
+	);
+
+	const closingParen = sourceCode.getLastToken(callExpression);
+
+	for (const [index, argument] of callExpression.arguments.map(
```

Comment:
You can just use `callExpression.arguments.entries()`.

Reply:
Thanks. Sorry for missing that.